.travis.yml

```diff
@@ -1,12 +1,11 @@
 language: node_js
 node_js:
-  - "4"
-  - "6"
-  - "8"
```

Comment:
I'm considering if tests should be rewritten to optionally run twice against `lib` and built `dist` files, conditionally based on runtime version. That would be entirely unrelated. For now, we'll continue to trust that babel does what it's supposed to do.